### ISSUE TITLE
Control for array of Strings does not work properly

### ISSUE THREAD

**Issue by koegel** [eclipsesource/jsonforms#678]
Resulting JSON is not an array of strings but an array of objects with the actual strings as property of the objects.

**Comment by ketysek** [comment]
So showing and editing the data below is not possible yet?
```
{
  "packages": [
    "flask",
    "requests"
  ]
}
```

I've tried to make it done this way:

**uischema**
```
{
  "type": "VerticalLayout",
  "elements": [
    {
      "type": "Control",
      "scope": "#/properties/packages"
    }
  ]
}
```

**schema:**
```
{
  "type": "object",
  "properties": {
    "packages": {
      "type": "array",
      "items": {
        "type": "string"
      }
    }
  }
}
```

but your seed app prints `No applicable field found.`

**Comment by edgarmueller** [comment]
@ketysek Yes, that's correct, but we are working on it. I've rescheduled it since I think this should be doable for the the next release.

**Comment by edgarmueller** [comment]
Fixed via #960, will be released with 2.0.1.
